web/src/message_fetch.js

```diff
@@ -469,6 +469,7 @@ export function initialize(home_view_loaded) {
             message_lists.home.select_id(data.anchor, {
                 then_scroll: true,
                 use_closest: true,
+                mark_read: false,
```

Comment:
This is probably something we want to do, but it isn't part of this particular issue. See https://github.com/zulip/zulip/issues/26021.